packages/core/parcel/src/cli.js

```diff
@@ -423,6 +435,15 @@ async function normalizeOptions(command): Promise<InitialParcelOptions> {
     command.detailedReport = '10';
   }
 
+  let additionalReporters = [
+    {packageName: '@parcel/reporter-cli', resolveFrom: __filename},
+    {packageName: '@parcel/reporter-dev-server', resolveFrom: __filename},
```

Comment:
Technically we could only add this for serve mode but it's probably fine either way